app/src/main/java/com/smlnskgmail/jaman/hashcheckerlite/logic/hashcalculator/ui/HashCalculatorFragment.java

```diff
@@ -97,6 +100,24 @@
             );
         } else {
             etGeneratedHash.setText(hashValue);
+            SharedPreferencesSettingsHelper settingsHelper = new SharedPreferencesSettingsHelper(context);
```

Comment:
You need to use `SettingsHelper` instance from Dagger injection. If a class don't contains `SettingsHelper` injection, add it as local public field with `@Inject` annotation.